database/001_initial_schema.sql

```diff
@@ -21,7 +21,7 @@ CREATE TABLE IF NOT EXISTS config (
 
 CREATE TABLE IF NOT EXISTS public.adaption (
   adaption_id SERIAL PRIMARY KEY NOT NULL,
-  adaption_type TEXT NOT NULL,
+  adaption_type TEXT NOT NULL UNIQUE,
```

Comment:
These changes does not make sense. Instead you should leave the data model as is. If you want to get adaption type in the with the log data, you can join this in.

SELECT foo, bar FROM system_log NATURAL JOIN adaption;

Reply:
It will also break the current datadump, so if you want to make this change, the datadump should be revised.

Reply:
Just ask me with what you are trying to solve, and i can help you with the database stuff (-:

Reply:
and also, adaption_type is not meant to be unique, think of it as an enum. We should probably have changed this to a while a go, but that would limit us from